UI/EnemyFileMenu.gd

```diff
@@ -355,6 +361,8 @@ func _do_save_file(file: File) -> void:
 				selected_string = enemy.custom_time
 			data.append(selected_string)
 
+			data.append(enemy.play_points)
```

Comment:
Remove this

Reply:
Won't it not be able to save the play points anymore if we remove this?

Reply:
A discussion on Discord determined that this should in fact stay.